cpp11/Ice/interceptor/InterceptorI.cpp

```diff
@@ -0,0 +1,42 @@
+//
+// Copyright (c) ZeroC, Inc. All rights reserved.
+//
+
+#include <InterceptorI.h>
+
+using namespace std;
+
+InterceptorI::InterceptorI(shared_ptr<Ice::Object> servant, shared_ptr<AuthenticatorI> authenticator,
+                           unordered_set<string> securedOperations) :
+    _servant(move(servant)), _authenticator(move(authenticator)), _securedOperations(move(securedOperations))
+{
+}
+
+bool
+InterceptorI::dispatch(Ice::Request& request)
+{
+    auto current = request.getCurrent();
+    //
+    // Check if the operation requires authorization to invoke.
+    //
+    if(_securedOperations.count(current.operation) > 0)
```

Comment:
Better use find which is more efficient with an ordered set.

Reply:
Right, find is what we want here, we don't need the count and I imagine count will always be a little more expensive since it requires iterating over the elements to count them.